libraries/Microsoft.Bot.Builder.Dialogs.Debugging/DebuggerSourceMap.cs

```diff
@@ -164,8 +165,25 @@ object IBreakpoints.ItemFor(Protocol.Breakpoint breakpoint)
             }
         }
 
-        private static bool PathEquals(string one, string two) =>
-            Path.Equals(Path.GetFullPath(one), Path.GetFullPath(two));
+        private static bool PathEquals(string one, string two)
+        {
+            var ext1 = Path.GetExtension(one).ToLower();
+
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                // we want to be case insensitive on windows
+                one = one.ToLower();
```

Comment:
Can people use different cultures for paths? If yes, `ToLowerInvariant()` is safer for cross culture strings in this case

Reply:
same on line 170